discovery-provider/src/app.py

```
@@ -196,17 +196,20 @@ def create_celery(test_config=None):
     global contract_addresses
     # pylint: enable=W0603
 
-    (
-        registry,
-        user_factory,
-        track_factory,
-        social_feature_factory,
-        playlist_factory,
-        user_library_factory,
-        user_replica_set_manager,
-        entity_manager,
-        contract_addresses,
-    ) = init_contracts()
+    try:
+        (
+            registry,
+            user_factory,
+            track_factory,
+            social_feature_factory,
+            playlist_factory,
+            user_library_factory,
+            user_replica_set_manager,
+            entity_manager,
+            contract_addresses,
+        ) = init_contracts()
+    except:
+        pass
```

when does this happen?  Can you add a comment about why we swallow this error?